finders.go

```diff
@@ -151,6 +144,26 @@ func (q *Query) All(models interface{}) error {
 	return nil
 }
 
+func (q *Query) paginate(models interface{}) error {
```

Comment:
@u007  We do: https://github.com/larrymjordan/pop/blob/45cae41ef8b64444dc8c2151de08bade2e110642/paginator.go#L97

@larrymjordan Maybe you should have a name a bit different here, to avoid confusion. WDYT?

Reply:
maybe modelPaginate